babel/src/test/resources/sql/big-query.iq

```diff
@@ -668,6 +668,77 @@ SELECT SAFE_ADD(CAST('NaN' AS DOUBLE), CAST(3 as BIGINT)) as NaN_result;
 (1 row)
 
 !ok
+
+#####################################################################
+# SAFE_DIVIDE
+#
+# SAFE_DIVIDE(value1, value2)
```

Comment:
Can we have some tests with some expression parameters?

`SAFE_DIVIDE(1*1, 1-1)`
`SAFE_DIVIDE(1, 1/9223372036854775807)`

Reply:
Will add these soon, good idea